tests/system/common.py

```diff
@@ -83,3 +84,22 @@ def wait_for_mesos_endpoint(timeout_sec=timedelta(minutes=5).total_seconds()):
     it returns false"""
 
     return shakedown.time_wait(lambda: shakedown.mesos_available_predicate(), timeout_seconds=timeout_sec)
+
+
+def metronome_available_predicate():
+    url = metronome_api_url()
+    try:
+        response = http.get(url)
+        return response.status_code == 200
+    except Exception as e:
+        return False
+
+def metronome_api_url():
+    return shakedown.dcos_url_path("/service/metronome/v1/jobs")
+
+
+def assert_one_job_run_with_one_task(client, job_id):
```

Comment:
Let's make it a bit more generic. How about `def assert_job_run(client, job_id, runs_number=1, tasks_number=1)`?